narwhals/_spark_like/expr.py

```diff
@@ -705,14 +705,43 @@ def fill_null(
         strategy: FillNullStrategy | None,
         limit: int | None,
     ) -> Self:
-        if strategy is not None:
-            msg = "Support for strategies is not yet implemented."
-            raise NotImplementedError(msg)
+        if strategy is not None:  # pragma: no cover
+            # TODO(Fbruzzesi): This is covered by pyspark, which however we do not run in
+            # the main CI suite. On the other hand, SQLFrame fails. We should figure out
+            # why and how to fix it, either upstream or from our side!
```

Comment:
outdated?

Reply:
Sorry I didn't have the time to comment before! No this is not fixed. CI is still xfailing for SQLFrame

Reply:
ah right - one more upstream issue https://github.com/eakmanrq/sqlframe/issues/374